jobs/azWebAppDeployJob.yml

```diff
@@ -232,6 +247,12 @@ jobs:
           steps:
           - checkout: self
 
+          - task: Bash@3
```

Comment:
As this is within the if statement would it not just log it as true every time?
Also there is a bash log of this parameter further up already